trigger/patch.go

```diff
@@ -273,15 +273,15 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 			Color:     slackColor,
 		})
 	}
-	var makespan time.Duration
-	if utility.IsZeroTime(t.patch.FinishTime) {
-		patchTasks, err := task.Find(task.ByVersion(t.patch.Id.Hex()))
-		if err == nil {
-			_, makespan = task.GetTimeSpent(patchTasks)
-		}
-	} else {
-		makespan = t.patch.FinishTime.Sub(t.patch.StartTime)
+
+	tasks, err := task.Find(task.ByVersion(t.patch.Id.Hex()))
+	if err != nil {
+		return nil, errors.Wrapf(err, "getting tasks for patch '%s'", t.patch.Id)
+	}
+	if tasks == nil {
+		return nil, errors.Wrapf(err, "no tasks found for patch '%s'", t.patch.Id)
 	}
+	_, makespan := task.GetFormattedTimeSpent(tasks)
```

Comment:
Do we only ever use the makespan value here, in which case should we just return that?

Reply:
And/or rather than having a separate function for this, GetTimeSpent could take in a formatted bool

Reply:
This makes it cleaner for the patch resolver, and it doesn't add much work. 
If you feel strongly about it I can change it though.

Reply:
(I missed the second comment, I'll switch to one function with a formatted bool)

Reply:
I actually can't change it to an isFormatted bool because GetTimeSpent() returns values as time.Duration, not strings.